### DISCUSSION TITLE
∀2 Retro Computing Analog VGA Card

### DISCUSSION THREAD

**Discussion by markadev** [markadev/AppleII-VGA#9] [Announcements]
There's an awesome fork of this project by David Kuder that completes a lot of the missing features of my firmware (like 80 column mode, text color setting, IIe support) and was featured on [Adrian's Digital Basement](https://youtu.be/ReTXGczq8Vk): https://github.com/V2RetroComputing/analog

It's a little more/less DIY-friendly (has a PCB available but uses a PAL & surface mount components) **but** it's not just a VGA card.  David has an alternate firmware that can emulate a Z80 Applicard and it looks like the hardware could be used to emulate _any_ Apple II peripheral card.  So if you're looking for a generic platform for building software-defined Apple II expansion cards then it's definitely worth a look.  Coupling this with the wifi-enabled Pico-W could be a game-changer for new Apple II peripherals.

It's also (sometimes) available for [purchase on tindie](https://www.tindie.com/products/v2retrocomputing/2-analog/) for a reasonable price if you can get one.

**Comment by epaell** [comment]
Funnily enough, this is the card (which I also saw on Adrian's Digital Basement YouTube clip) that brought me to your original design. I was initially tempted to take the easy route and just buy the card but not only was it unavailable but also didn't ship to my part of the world (Australia). Mostly likely, the shipping would also be some insane amount (I really don't know why that is the case for such small items. BTW, before seeing these designs I had previously considered the vidHD card but the shipping was US$182 for a US$135 card!). After searching I also came across the GitHub project you mentioned above but I'm not really set up to do SMD work (I might be able to program PALs but I haven't checked my programmer for those) - thankfully they pointed to your original design, and while not as flexible it did what I really wanted (quality video output) and used just garden-variety components.
